### ISSUE TITLE
Unwanted "Recovered Node" created for top-level @file nodes

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#896]
If a .leo file contains a clone of a top-level `@file` node, an unwanted and confusing Recovered Nodes tree is created for every cloned `@file` node. For example, in my leoPy.leo outline I have:
```
- Code (contains all @file nodes)
- Recent Files
  - @file leoGlobals.py (a clone of a previous node)
```
When I reload leoPy.leo I get the following Recovered Nodes tree:
```
- Recovered Nodes
  - Recovered node "@file leoGlobals.py" from leoGlobals.py
    - old:@file leoGlobals.py (empty)
    - new:@file leoGlobals.py (contains the entire text of the node, as if it were added)
```
If there are N cloned `@file` nodes, the "Recovered Nodes" node has N top-level children, one for each clone.

Rev 8ce10fab creates the new problem.  The comment in this rev is: leoCacher.readFile rewritten. An attempt to solve #892.

**Comment by vitalije** [comment]
Fixed at 8f9cc7148. Leo will not create recovered nodes for nodes that used to have empty body and no children.

**Comment by edreamleo** [comment]
Many thanks for this quick fix.  It works for me.
